### ISSUE TITLE
Local captcha verification

### ISSUE THREAD

**Issue by mimi89999** [christianwgd/django-friendly-captcha#5]
Hello,

I saw that there is an open source implementation of the verification server https://github.com/FriendlyCaptcha/friendly-lite-server. Would it be possible to implement it in this module to perform captcha verification locally?

**Comment by christianwgd** [comment]
I don't think this should be part of this package. But a far as i understand the docs, you can implement your own server as described and simply change the FRC_CAPTCHA_VERIFICATION_URL setting to call your service endpoint.
